docs/plugins/deploy.md

```diff
@@ -0,0 +1,35 @@
+# Deploy
+
+```
+serverless deploy --stage dev --region us-east-1
+```
+
+Deploys your service to AWS.
```

Comment:
hmmmm although it might be ok to leave it in here for now and we'll add the different subfolders when we implement azure